iFixit/react-commerce
# Add Links to Product to Upsell Section

## Overview

Lets have links to the product being upsold in the upsell section so that users can get more information about the product if they are interested.

## QA

Make sure that when you have something in your cart, the upsell section has links on the product name and on the product image. 

Now the entire upsell section is a link except the add to cart button basically.
~~Note: this links will be urls to the same domain that is serving the api responses, so if you are on localhost it will likely link you to some cominor domain.~~ 

Closes: #1249
